--- a/drivers/dma/Kconfig	2021-08-19 11:28:15.672921854 -0400
+++ b/drivers/dma/Kconfig	2021-08-19 11:28:35.029049568 -0400
@@ -91,7 +91,7 @@
 
 config AXI_DMAC
 	tristate "Analog Devices AXI-DMAC DMA support"
-	depends on MICROBLAZE || NIOS2 || ARCH_ZYNQ || ARCH_SOCFPGA || COMPILE_TEST
+	depends on MICROBLAZE || NIOS2 || ARCH_ZYNQ || ARCH_ZYNQMP || ARCH_SOCFPGA || COMPILE_TEST
 	select DMA_ENGINE
 	select DMA_VIRTUAL_CHANNELS
 	help
